### PR TITLE
Use ES6 syntax in List Editing UI tests

### PR DESCRIPTION
**Pull request by KsyushaM**

### CODE REVIEW

**Review by alexander-kotov-dx — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 1]
```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
...
-    LIST_ITEM_ICON_CONTAINER_CLASS = "dx-list-item-icon-container",
-    LIST_ITEM_CONTENT_CLASS = "dx-list-item-content",
-    LIST_ITEM_BEFORE_BAG_CLASS = "dx-list-item-before-bag";
+const LIST_ITEM_CLASS = "dx-list-item";
+const LIST_ITEM_ICON_CONTAINER_CLASS = "dx-list-item-icon-container";
+const LIST_ITEM_CONTENT_CLASS = "dx-list-item-content";
+const LIST_ITEM_BEFORE_BAG_CLASS = "dx-list-item-before-bag";
 
-var toSelector = function(cssClass) {
+const toSelector = (cssClass) => {
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]:
```suggestion
const toSelector = (cssClass) => `.${cssClass}`;
```

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 2]
```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
...
     return "." + cssClass;
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]:
```suggestion

```

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 3]
```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
...
 };
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]:
```suggestion

```

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 4]
```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
...
-    afterEach: function() {
+    afterEach: () => {
         delete decoratorRegistry.registry.menu.test;
     }
 });
 
-QUnit.test("positioning should be enabled while item prepared to delete", function(assert) {
-    var $list = $($("#templated-list").dxList({
+QUnit.test("positioning should be enabled while item prepared to delete", (assert) => {
+    const $list = $($("#templated-list").dxList({
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]:
```suggestion
    const $list = $("#templated-list").dxList({
```

I see other `$($(` entries below. Let's remove it.

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 5]
```diff
@@ -21,63 +21,62 @@ import "ui/action_sheet";
...
         items: ["0"],
         allowItemDeleting: true,
         itemDeleteMode: "test"
     }));
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]:
```suggestion
    });
```

File: testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js [hunk 6]
```diff
@@ -114,20 +114,19 @@ QUnit.test("active state should be enabled while item prepared to delete", funct
     }
 });
 
-QUnit.test("click should remove delete ready class", function(assert) {
-    var clickHandled = 0;
-
-    var $list = $("#templated-list").dxList({
+QUnit.test("click should remove delete ready class", (assert) => {
+    let clickHandled = 0;
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/listParts/editingUITests.js]: Use a stub instead of it and check `callCount`

**Review by DokaRus — COMMENTED** [review]
Let's do a little more refactoring?

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**